File: app/src/main/java/org/jellyfin/androidtv/data/model/ChapterItemInfo.java
package org.jellyfin.androidtv.data.model;

import org.jellyfin.apiclient.model.entities.ChapterInfo;

/**
 * Created by Eric on 2/12/2015.
 */
public class ChapterItemInfo extends ChapterInfo {
    private String itemId;

    public String getItemId() {
        return itemId;
    }

    public void setItemId(String itemId) {
        this.itemId = itemId;
    }
}

File: app/src/main/java/org/jellyfin/androidtv/data/model/ChapterItemInfo.kt
package org.jellyfin.androidtv.data.model

import org.jellyfin.androidtv.util.apiclient.JellyfinImage
import java.util.UUID

data class ChapterItemInfo(
	val itemId: UUID,
	val name: String?,
	val startPositionTicks: Long,
	val image: JellyfinImage?,
)
